package.json

```diff
@@ -84,6 +84,7 @@
     "karma-json-fixtures-preprocessor": "^0.0.6",
     "karma-ng-html2js-preprocessor": "^1.0.0",
     "karma-webpack": "^4.0.0-rc.3",
+    "loader-utils": "^1.2.3",
```

Comment:
Where's the `yarn.lock` change associated with this?

Reply:
@kevinlee12 `loader-utils` is a dependency for some of the packages that we're already using.
That's probably why `yarn.lock` wasn't updated.
https://github.com/oppia/oppia/blob/677896f1e7e2859d32312e9f3b190f4f59d61389/yarn.lock#L5531

Reply:
ah, okay. thanks for clarifying!